packages/native/examples/with-controlled-props/Search.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useContext, useEffect } from 'react';
+import {
+  SearchBox,
+  SearchContext
+} from '@appbaseio/react-native-searchbox';
+
+export default function Search({ setResetPagination }) {
```

Comment:
@anik-ghosh-au7 Where are you using `setResetPagination`?

Reply:
@bietkul https://github.com/appbaseio/searchbox/pull/105/files/8e725d69f293cae72a63483918d2439744b8751d#r676001024